docs/reference-guides/data/data-core-editor.md

```diff
@@ -1228,22 +1232,6 @@ _Usage_
 wp.data.dispatch( 'core/editor' ).editPost( { title: `${ newTitle }` } );
 ```
 
-```js
-// Get specific media size based on the featured media ID
-// Note: change sizes?.large for any registered size
-const getFeaturedMediaUrl = useSelect( ( select ) => {
-	const getFeaturedMediaId =
-		select( 'core/editor' ).getEditedPostAttribute( 'featured_media' );
-	const getMedia = select( 'core' ).getMedia( getFeaturedMediaId );
-
-	return (
-		getMedia?.media_details?.sizes?.large?.source_url ||
-		getMedia?.source_url ||
-		''
-	);
-}, [] );
-```
```

Comment:
This example looks like a copy/paste error to me, so I've deleted it. The same example was also present elsewhere (see the changes above this block).